qemu/tests/nic_opt.py

```diff
@@ -134,6 +134,9 @@ def test_ping():
     vm.verify_alive()
     session = vm.wait_for_serial_login()
     try:
+        output = vm.process.get_output()
+        if output != '':
```

Comment:
Could you check it at first? Sometimes the output of PPC platform will report some warning like `IRQ_XIVE capability must be present for KVM` but it's expected, so need to check if `error` or some other keywords exists.

Reply:
Hi @PaulYuuu I tested it on x86 before changing the code. It can correctly capture the corresponding error output: "qemu-kvm: unable to start vhost net: 14: falling back on userspace virtio".  If the PPC platform, there will be different output, I think the current usage is feasible. Please correct me if any mistakes. Thanks.